packages/connect/src/base/entry.ts

```diff
@@ -95,6 +98,24 @@ export class EntryNodes extends EventEmitter {
 
       this.options.publicNodes.removeListener('removePublicNode', this._onRemoveRelay as EntryNodes['onRemoveRelay'])
     }
+
+    this.stopDHTRenewal?.()
+  }
+
+  private startDHTRenewInterval() {
+    const renewDHTEntries = async function (this: EntryNodes) {
+      for (const usedRelay of this.usedRelays) {
+        const relay = relayFromRelayAddress(usedRelay)
+        const entry = this.availableEntryNodes.find((entry: EntryNodeData) => entry.id.equals(relay))
```

Comment:
Please rename `entry` in the lambda, so it does not clash with `entry` in the upper scope.